variants/backend-base/postcss.config.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable node/global-require */
+'use strict';
+
+module.exports = {
+  plugins: [
+    require('postcss-import'),
+    require('postcss-flexbugs-fixes'),
+    require('postcss-preset-env')({
+      autoprefixer: {
+        flexbox: 'no-2009'
+      },
+      stage: 3
+    })
+  ]
+};
```

Comment:
@G-Rath I did hit errors completing a `bundle exec rails webpacker:compile` in the bootstrap variant without this file.